packages/preferences/src/browser/ui/settings-ui-plus-widget.tsx

```diff
@@ -0,0 +1,46 @@
+/********************************************************************************
+ * Copyright (C) 2020 Ericsson and others.
```

Comment:
Do you work for Ericsson?

Reply:
Yes. This package started as part EMCA Studio for Ericsson, before we began the process of upstreaming it.